app/assets/stylesheets/_orders.scss

```diff
@@ -3,3 +3,54 @@
   margin:0 auto;
   margin-bottom: 30px;
 }
+
+.admin-order-title{
+  text-align:center;
```

Comment:
Properties should be ordered margin-bottom, text-align<br>Colon after property should be followed by one space